blueflood-core/src/main/java/com/rackspacecloud/blueflood/io/datastax/DatastaxShardStateIO.java

```diff
@@ -54,7 +54,7 @@
             Select.Where statement = QueryBuilder
                     .select()
                     .all()
-                    .from(CassandraModel.QUOTED_KEYSPACE, CassandraModel.CF_METRICS_STATE_NAME)
+                    .from(CassandraModel.CF_METRICS_STATE_NAME)
```

Comment:
I put it  https://github.com/usnavi/blueflood/blob/metadata_datastax2/blueflood-core/src/main/java/com/rackspacecloud/blueflood/io/datastax/DatastaxIO.java#L82